### ISSUE TITLE
HPC Carpentry

### ISSUE THREAD

**Issue by bkmgit** [carpentries-incubator/lesson-parallel-python#84]
Might this lesson be something that could be used in an [HPC Carpentry](https://www.hpc-carpentry.org/) workshop?

**Comment by jhidding** [comment]
Yes, we sometimes teach this material as part of a larger HPC workshop.
